jwst/datamodels/tests/test_fits.py

```diff
@@ -68,3 +69,24 @@ def test_casting():
         total = dm.data.sum()
         dm.data = dm.data + 2
         assert dm.data.sum() > total
+
+
+@pytest.mark.skip(reason='fix required for asdf')
+def test_resave_duplication_bug(tmp_path):
+    """
+    An issue in asdf (https://github.com/asdf-format/asdf/issues/1232)
+    resulted in duplication of data when a model was read from and then
+    written to a fits file.
```

Comment:
You can mention something about the numpy arrays get written to the `hdu.data` members and remained stored in the asdf tree bundled in the ASDF extension.